test/http.spec.ts

```diff
@@ -2,56 +2,80 @@ import { deepEqual, equal, ok } from "assert";
 import { HTTPError, RequestError } from "../lib/exceptions";
 import HTTPClient from "../lib/http";
 import { getStreamData } from "./helpers/stream";
-import { close, listen } from "./helpers/test-server";
+import * as nock from "nock";
 import { readFileSync, createReadStream } from "fs";
 import { join } from "path";
 
 const pkg = require("../package.json");
-
-const TEST_PORT = parseInt(process.env.TEST_PORT, 10);
-
-const getRecentReq = (): any =>
-  JSON.parse(readFileSync(join(__dirname, "helpers/request.json")).toString());
+const baseURL = "https://line.me";
+const defaultHeaders = {
+  "test-header-key": "Test-Header-Value",
+};
 
 describe("http", () => {
   const http = new HTTPClient({
-    baseURL: `http://localhost:${TEST_PORT}`,
-    defaultHeaders: {
-      "test-header-key": "Test-Header-Value",
-    },
+    baseURL,
```

Comment:
```
new HTTPClient({
      baseURL
})
```
Since we don't need this param any more. Maybe it's better to remove `baseURL` and make tests based on different endpoints.

Reply:
Alright, but maybe it should be a new pull request?